runtime/onert/core/src/dumper/text/GraphDumper.cc

```diff
@@ -87,8 +87,10 @@ void dumpGraph(const ir::Graph &graph)
     const auto &op = graph.operations().at(op_ind);
     VERBOSE(GraphDumper) << "  " << formatOperation(op, op_ind) << "\n";
   }
+  graph.operands().iterate([&](const ir::OperandIndex &idx, const ir::Operand &oprd) {
+    VERBOSE(GraphDumper) << "  Origin(" << idx << "): " << oprd.originIndex() << std::endl;
+  });
```

Comment:
It helps to track Operands information after each phases. It helps to find the problem like https://github.com/Samsung/ONE/pull/12223/files#r1416719944.